### DISCUSSION TITLE
Assign Label for each ingredient  in receipe

### DISCUSSION THREAD

**Discussion by jjam8644** [mealie-recipes/mealie#3946] [Ideas]
Could we please assign a label for each ingredient. That way you would not have to assign a label every time you add recipes to the shopping list.

**Comment by michael-genson** [comment]
It sounds like you aren't parsing your foods or using the food store:

https://docs.mealie.io/documentation/getting-started/faq/#how-do-i-enable-smart-ingredient-handling

If you have a food store with labels, then those ingredients are automatically labeled in the shopping list

**Comment by jjam8644** [comment]
You are right. Thank you!
